Kernel/TTY/TTY.h

```diff
@@ -29,6 +29,7 @@ class TTY : public CharacterDevice {
     void set_termios(const termios&);
     bool should_generate_signals() const { return m_termios.c_lflag & ISIG; }
     bool should_echo_input() const { return m_termios.c_lflag & ECHO; }
+    //bool should_echo_control() const { return m_termios.c_lflag & ECHOCTL; }
```

Comment:
Either uncomment this or remove it. It's okay to have to-be-used-in-the-future things, but let's not have commented-out APIs :)